_layouts/base.html

```diff
@@ -25,6 +25,7 @@
     <script src="/js/bootstrap-3.3.7.min.js"></script>
     <script src="/js/d3.v4.min.js"></script>
     <script src="/js/main.js"></script>
+    <script src="http://localhost/openup/cross-project-banner/openup-banner.js"></script>
```

Comment:
Thanks. That was to test the cross project footer with the logo, I accidentally left that in there.